test/nodejs/errorreporting/errorreporting.js

```diff
@@ -0,0 +1,255 @@
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+'use strict';
+
+var async = require('async');
+
+if (process.argv.length <= 2) {
+  console.log('Usage: %s <project_id> <service> <version>', __filename);
+  process.exit(1);
+}
+
+var PROJECT_NAME = process.argv[2];
+var SERVICE = process.argv[3];
+var VERSION = process.argv[4];
+var USER_NAME = 'testuser';
+
+var clouderrorreportingV1beta1 =
+    require('@google-cloud/errorreporting').v1beta1();
+
+/**
+ * Integration test for ReportErrorsServiceApi.reportErrorEvent
+ *
+ * Report first two error events (to be one group) and third event
+ * (to be another group)
+ */
+function createErrorEvents(done) {
+  var api = clouderrorreportingV1beta1.reportErrorsServiceApi();
+  var formattedProjectName = api.projectPath(PROJECT_NAME);
+
+  var httpRequestConext = {
+    method: 'GET',
+    url: 'https://20160920t135819-dot-gapic-test.appspot.com'
+  };
+  var sourceLocation1 = {
+    file_path: 'foo/bar',
+    line_number: 20,
+    function_name: 'doGet'
+  };
+  var sourceLocation2 = {
+    file_path: 'foo/bar',
+    line_number: 30,
+    function_name: 'doSet'
+  };
+  var event1 = {
+    message: 'error reporting test event1',
+    service_context: {
+      service: SERVICE,
+      version: VERSION
+    },
+    context: {
+      http_request: httpRequestConext,
+      user: USER_NAME,
+      report_location: sourceLocation1
+    }
+  };
+  var event2 = {
+    message: 'error reporting test event2',
+    service_context: {
+      service: SERVICE,
+      version: VERSION
+    },
+    context: {
+      http_request: httpRequestConext,
+      user: USER_NAME,
+      report_location: sourceLocation1
+    }
+  };
+  var event3 = {
+    message: 'error reporting test event3',
+    service_context: {
+      service: SERVICE,
+      version: VERSION
+    },
+    context: {
+      http_request: httpRequestConext,
+      user: USER_NAME,
+      report_location: sourceLocation2
+    }
+  };
+
+  // report the three events in parallel
+  async.eachOf([event1, event2, event3], function(event, index, callback) {
+    api.reportErrorEvent(formattedProjectName, event, function(err, response) {
+      console.log('Reporting error event %d', index + 1);
+      if (err) {
+        console.error(err);
+      }
+      callback(err);
+    });
+  }, function(err) {
+    if (err) {
+      console.error(err);
+    }
+    console.log('createErrorEvents Done');
```

Comment:
`else { console.log('createErrorEvents Done'); }` ? Otherwise, it will be printed out when errors (did you intend that?)